docs/api/controller.md

```diff
@@ -109,20 +109,20 @@ By default, the node will be included securely (with encryption) if a network ke
 
 ### Managing associations
 
-The following methods can be used to manage associations between nodes. This only works AFTER the interview process!
+The following methods can be used to manage associations between nodes and/or endpoints. This only works AFTER the interview process!
 
 ```ts
-getAssociationGroups(nodeId: number): ReadonlyMap<number, AssociationGroup>;
-getAssociations(nodeId: number): ReadonlyMap<number, readonly Association[]>;
-isAssociationAllowed(nodeId: number, group: number, association: Association): boolean;
-addAssociations(nodeId: number, group: number, associations: Association[]): Promise<void>;
-removeAssociations(nodeId: number, group: number, associations: Association[]): Promise<void>;
+getAssociationGroups(nodeId: number, endpointIndex?: number): ReadonlyMap<number, AssociationGroup>;
+getAssociations(nodeId: number, endpointIndex: number): ReadonlyMap<number, readonly AssociationAddress[]>;
```

Comment:
@AlCalzone Is endpoint required now when calling getAssociations?

Reply:
It shouldn't. I probably forgot a `?` in the docs.

Reply:
Yep, forgot a `?`. Returning all associations of the node and all endpoints would be breaking, so I'll have to add another method for that.